src/Microsoft.Data/StringColumn.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data
+{
+    public partial class StringColumn : BaseColumn
```

Comment:
public classes should at least have an xml `summary` doc comment explaining what they are at a high level.